doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example

```diff
@@ -13,6 +13,7 @@ turncredentials = {
 
 cross_domain_bosh = false;
 consider_bosh_secure = true;
+-- https_ports = { }; -- Remove this line to prevent listening on port 5284
```

Comment:
I wonder if we should just have it uncommented, thus disabled by default. Thoughts @damencho ?

Reply:
Well, maybe yeah ... Anyway, we started modifying global configs ... The dangerous thing is that if somebody had prosody already with some config we can break it ...

Reply:
Let's leave the comment then.